components/style/values/specified/font.rs

```diff
@@ -988,6 +989,113 @@ impl From<FontSynthesis> for u8 {
     }
 }
 
+#[derive(Clone, Debug, Eq, MallocSizeOf, PartialEq)]
+/// Allows authors to explicitly specify the language system of the font,
+/// overriding the language system implied by the content language
+pub enum FontLanguageOverride {
+    /// When rendering with OpenType fonts,
+    /// the content language of the element is
+    /// used to infer the OpenType language system
+    Normal,
+    /// Single three-letter case-sensitive OpenType language system tag,
+    /// specifies the OpenType language system to be used instead of
+    /// the language system implied by the language of the element
+    Override(String),
```

Comment:
If you feel like it, you can use `Box<str>` to save some memory (using `into_boxed_string`) during parsing.

Reply:
Thanks! I'll try it! I'd like to try `Box<str>`!